.eslintrc.js

```diff
@@ -344,6 +345,7 @@ module.exports = {
         'packages/kbn-plugin-helpers/**/*.js',
         'packages/kbn-eslint-import-resolver-kibana/**/*.js',
         'packages/kbn-eslint-plugin-eslint/**/*',
+        'packages/kbn-utility-types/**/*',
```

Comment:
Lets remove this, this is for untranspiled JS

Reply:
Done.